test/fixtures/results/oas-results.ts

```diff
@@ -2,32 +2,68 @@ import { OASResult } from '../../../src/results';
 import {
   operationExampleResultFailuresWarnings,
   operationExampleResultFailuresWarningsString,
+  operationExampleResultFailuresWarningsStructure,
   operationExampleResultFailuresNoWarnings,
   operationExampleResultFailuresNoWarningsString,
   operationExampleResultNoFailuresWarnings,
   operationExampleResultNoFailuresWarningsString,
+  operationExampleResultNoFailuresWarningsStructure,
+  operationExampleResultMixedStructure,
 } from './operation-example-results';
+import { StructuredOutput } from '../../../src/utilities/structured-output';
+import { OASSecurityScheme } from '../../../src/utilities/oas-security';
 
 export const oasResultSuccess = new OASResult(
   'winterfell',
+  undefined,
+  undefined,
+  [],
   [operationExampleResultNoFailuresWarnings],
   undefined,
 );
 
 export const oasResultSuccessString = `winterfell: Succeeded
 ${operationExampleResultNoFailuresWarningsString}`;
 
+export const oasResultSuccessStructure: StructuredOutput = {
+  id: oasResultSuccess.testName,
+  config: {
+    oasPath: String(oasResultSuccess.oasPath),
+    server: String(oasResultSuccess.server),
+    authenticationType: '',
+  },
+  error: undefined,
```

Comment:
Is error being undefined equivalent to it not being in the object at all? I think it might be nice to not include it if it's undefined

Reply:
Yes, they are equivalent.  I set `undefined` values explicitly in a few places in this file because it mirrors the corresponding `OASResult` constructor calls and seemed to made things easier to read.